### PR TITLE
Support multiple versions of the helm client

### PR DESCRIPTION
**Pull request by andreas-kupries**
Ref https://trello.com/c/ZLlm1pbe/933-minibroker-brain-test-for-redis-fails-due-to-client-mismatch

Package all versions needed by vagrant, our CI, an CaaSP3.
Detect and choose version to use at runtime (brain tests).

### CODE REVIEW

File: src/scf-release/packages/helm/packaging [hunk 1]
```diff
@@ -2,13 +2,28 @@
...
-curl -L -o helm.tar.gz "https://storage.googleapis.com/kubernetes-helm/helm-${VERSION}-linux-amd64.tar.gz"
+echo Packaging helm cli versions: ${VERSIONS}
 
-BIN_DIR="${BOSH_INSTALL_TARGET}/bin"
+for version in $VERSIONS
+do
+    DEST="${BIN_DIR}/${version}"
+
+    echo Retrieving helm cli "${version}"
+    curl -L -o helm.tar.gz "https://storage.googleapis.com/kubernetes-helm/helm-v${version}-linux-amd64.tar.gz"
```
[the remark this answers is not in the document]
Comment (andreas-kupries) [reply src/scf-release/packages/helm/packaging]: The packaging script runs in a container whose whole filesystem is temporary. None of the packaging scripts do anything with temp files IIRC. No need to double temp it ;)

File: src/scf-release/jobs/acceptance-tests-brain/templates/run.erb [hunk 2]
```diff
@@ -2,11 +2,31 @@
 
 set -o errexit
 
+# Use the primary linked helm client to determine the actual version
+# of the server. Check if we have a client for that version packaged,
+# and if yes, put it on the PATH. Else fails with a nice message.
+
+HBASE=/var/vcap/packages/helm/bin
```
[the remark this answers is not in the document]
Comment (andreas-kupries) [reply src/scf-release/jobs/acceptance-tests-brain/templates/run.erb]: Will change.

File: src/scf-release/jobs/acceptance-tests-brain/templates/run.erb [hunk 3]
```diff
@@ -2,11 +2,31 @@
...
+HELM_VERSION="$(${HBASE}/helm version| grep Server | sed -e 's/.*"v//' -e 's/".*//')"
```
[the remark this answers is not in the document]
Comment (andreas-kupries) [reply src/scf-release/jobs/acceptance-tests-brain/templates/run.erb]: Heh. Ok, will change.

[the remark this answers is not in the document]
Comment (andreas-kupries) [reply src/scf-release/jobs/acceptance-tests-brain/templates/run.erb]: Done with commit dc190ec81b54f6fdfa64105449664bda089bd71d

File: src/scf-release/jobs/acceptance-tests-brain/templates/run.erb (hunk 2 above)
[the remark this answers is not in the document]
Comment (andreas-kupries) [reply src/scf-release/jobs/acceptance-tests-brain/templates/run.erb]: Done with commit dc190ec81b54f6fdfa64105449664bda089bd71d

File: src/scf-release/packages/helm/packaging (hunk 1 above)
[the remark this answers is not in the document]
Comment (f0rmiga) [reply src/scf-release/packages/helm/packaging]: ahh cool

**Pull request merged**